src/app/modules/core/services/mock.service.ts

```diff
@@ -6,15 +6,19 @@ import {
   REQUEST_TYPE_DELETE,
 } from './api.service';
 import { Injectable } from '@angular/core';
-import { ApiBaseUrl } from '../constants/appConstants';
 import { Observable } from 'rxjs';
 import { UserData } from '../interfaces/user-data';
+import { StateCity } from '../interfaces/state-city';
 
 @Injectable()
 export class MockService {
   constructor(public apiService: ApiService) {}
 
   public getUserList(): Observable<Array<UserData>> {
```

Comment:
Do we need this method? Remove all the unwanted methods

Reply:
getUserList() method is used in ContactList Component.
MockService is used in ContactList as well Profile Component